src/eduid/satosa/scimapi/accr.py

```diff
@@ -63,21 +64,22 @@ def process(self, context: satosa.context.Context, data: satosa.internal.Interna
 
             if not supported_accr_to_forward:
                 raise SATOSAAuthenticationError(context.state, "Unsupported ACCR")
-            else:
-                requested_accr = supported_accr_to_forward
+
+            requested_accr = supported_accr_to_forward.copy()
```

Comment:
to be sure that later changes to the list supported_accr_to_forward does not reflect to requested_accr I would use deepcopy().